index.d.ts

```diff
@@ -70,6 +77,30 @@ declare const findUp: {
 	/**
 	Return this in a `matcher` function to stop the search and force `findUp` to immediately return `undefined`.
 	*/
+	exists(path: string): boolean;
```

Comment:
I think having `findUp.exists` be a sync function is inconsistent with the rest of the module, everything else is async first so `findUp.exists` should return `Promise<boolean>`.  This comment applies to all functions being added.

I don't know if the sync version should be `findUp.sync.exists` or `findUp.exists.sync`.  I lean towards `findUp.sync.exists`, this way someone could use:
```js
const findUp = require('find-up').sync;
findUp.exists('some-file.js');
```

Reply:
> I lean towards findUp.sync.exists, this way someone could use:

👍